src/Microsoft.ServiceFabric.Services.Queryable/Controller/ValueViewModel.cs

```diff
@@ -5,6 +5,8 @@ namespace Microsoft.ServiceFabric.Services.Queryable.Controller
 {
 	public class ValueViewModel
 	{
+		public string Operation { get; set; }
```

Comment:
You should make Operation an enum.  The front-end just has to send strings just like it does.  Json (Newtonsoft) will deserialize it correctly.

public enum Operation
{
     Add,
     Update,
     Delete,
}